src/sourmash/sbt.py

```diff
@@ -378,13 +383,21 @@ def search(self, query, *args, **kwargs):
         if do_containment:
             search_fn = search_minhashes_containment
             query_match = lambda x: tree_query.contained_by(x, downsample=True)
+        elif do_max_containment:
+            search_fn = search_minhashes_max_containment
+            query_match = lambda x: tree_query.max_containment(x,
+                                                               downsample=True)
 
         if best_only:            # this needs to be reset for each SBT
             search_fn = SearchMinHashesFindBest().search
```

Comment:
does this need to change at all for containment?

I see jaccard over here? https://github.com/dib-lab/sourmash/blob/ca201cfc1824900bfb94aaadd94e163c70b7cf4e/src/sourmash/sbtmh.py#L135

Reply:
good catch, this is straight up a bug - when `--best-only` is specified, it reverts to calculating Jaccard similarity on SBTs!

I added an exception raise and some tests in 108748a4 to prevent this for now, will punt this to a new issue to fix properly.